### PR TITLE
Local and Remote Options were added to the IOC

### PR DESCRIPTION
**Pull request by danielmaclaren**
- https://github.com/ISISComputingGroup/IBEX/issues/7878

### CODE REVIEW

File: aeroflexSup/aeroflex_common.db [hunk 1]
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
     field(ONAM, "COMMS DISABLED")
 }
 
+record(bi, "$(P)CMODE_RBV") {
```
Comment (FreddieAkeroyd) [inline aeroflexSup/aeroflex_common.db]: AS per https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#signal-qualifiers this should be just `CMODE` with optionally an alias to `CMODE:SP:RBV`, this naming convention helps setting blocks work from python as you set a block to "CMODE" and it adds `:SP` if you write from python

File: aeroflexSup/aeroflex_common.db [hunk 2]
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
...
+record(bo, "$(P)CMODE:SP") 
+{
+    field(DTYP, "Raw Soft Channel")
+    field(DESC, "Enable Local or Remote Mode")
+    field(ZRST, "REMOTE")
+    field(ONST, "LOCAL")
+	field(ZRVL, "0")
+	field(ONVL, "1")  
+    field(OUT, "$(P)DISABLE PP")
+    field(FLNK, "$(P)CMODE_RBV.PROC")
```
Comment (FreddieAkeroyd) [inline aeroflexSup/aeroflex_common.db]: FLNK not needed as CMODE is CP

File: aeroflexSup/aeroflex_common.db [hunk 3]
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
...
+}
+
+record(bo, "$(P)CMODE:SP") 
+{
+    field(DTYP, "Raw Soft Channel")
+    field(DESC, "Enable Local or Remote Mode")
+    field(ZRST, "REMOTE")
+    field(ONST, "LOCAL")
+	field(ZRVL, "0")
+	field(ONVL, "1")
```
Comment (FreddieAkeroyd) [inline aeroflexSup/aeroflex_common.db]: Does this load OK? I think bo used ZNAM/ONAM instead

Comment (esmith1729) [reply aeroflexSup/aeroflex_common.db]: Changed to ZNAM and ONAM.

File: aeroflexSup/aeroflex_common.db (hunk 2 above)
Comment (esmith1729) [reply aeroflexSup/aeroflex_common.db]: Removed FLINK

File: aeroflexSup/aeroflex_common.db (hunk 1 above)
Comment (esmith1729) [reply aeroflexSup/aeroflex_common.db]: Changed PV name to "CMODE", and added alias for SP:RBV as well

**Pull request merged**